cpp/src/Ice/ConnectionFactory.cpp

```diff
@@ -347,9 +349,17 @@ IceInternal::OutgoingConnectionFactory::OutgoingConnectionFactory(
     : _communicator(communicator),
       _instance(instance),
       _monitor(new FactoryACMMonitor(instance, instance->clientACM())),
+      _idleTimeout(60000ms),
```

Comment:
We can also do, which seems easier to read.
```suggestion
      _idleTimeout(60s),
```